autoarray/inversion/mappers.py

```diff
@@ -255,29 +285,28 @@ def __init__(self, grid, pixelization_grid, hyper_image=None):
             A pre-computed hyper-image of the image the mapper is expected to reconstruct, used for adaptive analysis.
         """
         super(MapperVoronoi, self).__init__(
-            grid=grid, pixelization_grid=pixelization_grid, hyper_image=hyper_image
+            source_full_grid=source_full_grid,
+            source_pixelization_grid=source_pixelization_grid,
+            data_pixelization_grid=data_pixelization_grid,
+            hyper_image=hyper_image,
         )
 
     @property
-    def is_image_plane_pixelization(self):
-        return True
-
-    @property
-    def pixelization_1d_index_for_sub_mask_1d_index(self):
+    def pixelization_1d_index_for_sub_full_1d_index(self):
         """  The 1D index mappings between the sub pixels and Voronoi pixelization pixels. """
 
         return mapper_util.pixelization_1d_index_for_voronoi_sub_mask_1d_index_from(
-            grid=self.grid,
-            nearest_pixelization_1d_index_for_mask_1d_index=self.pixelization_grid.nearest_pixelization_1d_index_for_mask_1d_index,
-            mask_1d_index_for_sub_mask_1d_index=self.grid.regions._mask_1d_index_for_sub_mask_1d_index,
-            pixelization_grid=self.pixelization_grid,
-            pixel_neighbors=self.pixelization_grid.pixel_neighbors,
-            pixel_neighbors_size=self.pixelization_grid.pixel_neighbors_size,
+            grid=self.source_full_grid,
+            nearest_pixelization_1d_index_for_mask_1d_index=self.source_pixelization_grid.nearest_pixelization_1d_index_for_mask_1d_index,

```

Comment:
Some of these names are way too long.

I prefer shorter names distinguished by context

Reply:
This module is another desperately needing a refactor. Its on my to do list.